android/assets/jsons/translations/template.properties

```diff
@@ -990,6 +990,13 @@ Found Pantheon =
 Follow [belief] = 
 Religions and Beliefs = 
          
+# Religion overview screen
+Religion Name: = 
+Founding Civ: = 
+??? = ???
```

Comment:
This really doesn't need translation. I've been using a single question mark for the random for years and translators have picked up on all kinds of really esoteric things, but the question mark appears to be universal.